sdks/java/extensions/sql/zetasql/src/main/java/org/apache/beam/sdk/extensions/sql/zetasql/translation/ConversionContext.java

```diff
@@ -33,7 +33,16 @@
   private final ExpressionConverter expressionConverter;
   private final RelOptCluster cluster;
   private final QueryTrait trait;
-  private final Map<List<String>, ResolvedNode> sqlUDTVF;
+  // Pure-sql user-defined table-valued function can be resolved by Analyzer. Keeping the
```

Comment:
I'm guessing the meaning of `Pure-sql` might not be clear to those who are unfamiliar with our plans to add cross-language UDF.

Reply:
The comments here are all about "pure-sql" UDF (it won't apply to java/python UDF). Is there a better naming in your mind (or you are prefer to add a link some where to explain what is "pure-sql" UDF?)

Reply:
Perhaps "native SQL"/"SQL native" functions is a more apt terminology.

Reply:
Adopted "SQL native".